theano/sandbox/cuda/tests/test_opt.py

```diff
@@ -86,6 +87,176 @@ def test_gpualloc():
     assert numpy.any([isinstance(x.op, cuda.GpuAlloc) for x in l])
 
 
+class Test_local_elemwise_alloc(unittest.TestCase):
```

Comment:
Duplicating the test isn't good. Can you inherit from the original test in tensor/tests/test_opt.py and override the assert fct? Was there other change? You can see an example in cuda/tests/test_basic_ops.py with the class T_Join_and_Split